src/api/app/components/notification_action_description_component.rb

```diff
@@ -54,4 +56,16 @@ def bs_request
                       @notification.notifiable.commentable
                     end
   end
+
+  def event_workflow_run_fail
```

Comment:
We're asking `WorkflowRun` a lot of data. I think this method should be moved there.